### ISSUE TITLE
Missing controls and captions not showing

### ISSUE THREAD

**Issue by ellinika** [ableplayer/ableplayer#466]
Hi,

I set up Able Player just like Demo 3 here: https://ableplayer.github.io/ableplayer/demos/video3.html

However, the player doesn't look like the demo. The controls are missing and the captions are not displaying. Any ideas on how to fix this issue?

Here's a screenshot of the problem:
![able-player-missing-controls](https://user-images.githubusercontent.com/85311255/120693621-d68ddc80-c46e-11eb-8714-9aecb6cd4339.jpg)

Here's my code: 
`<script src="//ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
<script src="/wp-content/plugins/able-player/thirdparty/js.cookie.js"></script>
<link rel="stylesheet" href="/wp-content/plugins/able-player/build/ableplayer.min.css" type="text/css"/>
<link rel="stylesheet" href="/wp-content/plugins/able-player/styles/ableplayer.css" type="text/css">
<link rel="stylesheet" href="/wp-content/plugins/able-player/stylesheets/stylesheet.css" type="text/css">
<script src="/wp-content/plugins/able-player/build/ableplayer.js"></script>`

video id="our-house-video" poster="../wp-content/uploads/2020/09/My-Place-scaled.jpg" preload="auto" playsinline data-able-player data-skin="2020"
source type="video/mp4" src="../wp-content/uploads/videos/our-house-v4.mp4" 
track kind="captions" src="../wp-content/uploads/videos/our-house.vtt" srclang="en" label="English captions" 
    track kind="descriptions" src="../wp-content/uploads/videos/our-house.vtt" srclang="en" 
video

**Comment by ellinika** [comment]
<video id="our-house-video" poster="../wp-content/uploads/2020/09/My-Place-scaled.jpg" preload="auto" playsinline data-able-player data-skin="2020">
<source type="video/mp4" src="../wp-content/uploads/videos/our-house-v4.mp4" />
<track kind="captions" src="../wp-content/uploads/videos/our-house.vtt" srclang="en" label="English captions" />
    <track kind="descriptions" src="../wp-content/uploads/videos/our-house.vtt" srclang="en" />
  </video>

**Comment by dingq84** [comment]
I ran into the same problem, and I found that the reason was that the ableplayer script was executed before the video element rendered. I solved this problem by modifying document.ready event to window.load event in ableplayer.js below.
![image](https://user-images.githubusercontent.com/32992580/120883521-44bdd500-c610-11eb-8675-35401f9ea90f.png)

**Comment by terrill** [comment]
Thanks for the tip @dingq84 .  Typically if Able Player fails to load, there will be an error message within the Console in your browser's developer tools, which will help you troubleshoot.  I'm closing this issue for now.  If this is still a problem @ellinika (or if it happens again) feel free to comment and we can reopen the issue and troubleshoot.
